client/my-sites/plugins/plugin-details.jsx

```diff
@@ -237,7 +238,14 @@ function PluginDetails( props ) {
 				} )
 			);
 		}
-	}, [ fullPlugin.name, props.pluginSlug, selectedSite, localizePath ] );
+	}, [ dispatch, translate, localizePath, selectedSite?.slug, fullPlugin.name, props.pluginSlug ] );
+
+	useServerEffect( setBreadcrumbs );
```

Comment:
The server never calls the `useEffect` hook, and thus never sets the breadcrumbs items. Calling the breadcrumb functionality through `useServerEffect` allows us to only run it on the server and leave the client flow untouched.